app/policies/hbx_profile_policy.rb

```diff
@@ -411,10 +411,6 @@ def edit?
     end
   end
 
-  def inbox?
-    index?
-  end
-
```

Comment:
This was a duplicate method, there is another `inbox?` method on line 262 of this file which was the exact same